corehq/util/workbook_reading/tests/files/csv/types.csv

```diff
@@ -11,5 +11,5 @@ Bool-T,TRUE
 Empty,
 Percent,49%
 Calculation,2
-Styled,Sطαйλד
+Styled,\u0053\u0637\u03B1\u0439\u03BB\u05D3\u0020\u0074\u044D\u03BE\u0163
```

Comment:
Is it a standard CSV formatting convention to use unicode escapes rather than UTF-8 encoded characters? I think this change should be reverted.

Reply:
I don't think it is, but I included it just in case the situation came up where a CSV had those characters. I think removing the unicode escapes here also removes the need for `.decode('unicode-escape')` in `corehq/util/workbook_reading/adapters/csv.py `. I'm guessing it's pretty unlikely that a CSV will have unicode escapes, so I'll change this line to `Styled,Sطαйλד tэξţ`